### DISCUSSION TITLE
README displays markdown not properly

### DISCUSSION THREAD

**Discussion by karinzaa** [community/community#110619] [Repositories]
If you used HTML tag in README markdown it bug to display.
 Example: https://github.com/karinzaa/JunctionKanshi/blob/main/README.md
<img width="797" alt="image" src="https://github.com/community/community/assets/25452435/00281c5e-9b27-4a5c-9a9b-45a416b6262d">

**Comment by ItsJoshBrown** [comment]
If you are trying to get it a the top and centered on your markdown, you can do something like this, and add a little in-line CSS. 
and then you can remove `<p align="center">`

```HTML
<br>
<img src="https://github.com/karinzaa/JunctionKanshi/blob/main/images/JunctionKanshiLogoOriginal.png" style="display: block; margin-left: auto; margin-right: auto; width: 350px;" alt="JunctionKansh Logol">
</br>
```	

Example: 

<img width="462" alt="image" src="https://github.com/community/community/assets/64455452/25d5f354-8f26-48f0-be06-9e196cd33ad1">

**Reply by karinzaa** [reply]
I would like The logo is at the top right on the desktop.

**Reply by ItsJoshBrown** [reply]
I see, so you would want to do something like this then. probably not the best implementation but it works 😄 

```HTML
<br>
<img src="https://github.com/karinzaa/JunctionKanshi/blob/main/images/JunctionKanshiLogoOriginal.png" style="display: block; margin-left: auto; margin-right: 0; width: 350px;" alt="JunctionKansh Logol">
</br>
```

**Reply by ItsJoshBrown** [reply]
You are right, it is bugged out, I'm sorry that's not working in Github. only in markdown viewer weird. 

I was able to sort of git in a janky way. just add more breaks for spacing 

```HTML
<br>
<img style="display: block;" align="right" width="350" src="https://github.com/karinzaa/JunctionKanshi/blob/main/images/JunctionKanshiLogoOriginal.png">
<br><br><br><br><br><br>
```

**Comment by IAmVigneswaran** [comment]
Yes, I also experiencing the same issue.

https://github.com/TheAcharya/csv2notion-neo
https://github.com/TheAcharya/MarkersExtractor

Hope this can be fixed.

Thank you.

**Comment by IAmVigneswaran** [comment]
It has been several weeks.

Why is this issues has not been addressed and fixed?

Thank you.

**Comment by smartdev328** [comment]
Markdown is a lightweight markup language with plain-text formatting syntax designed to be converted into HTML. When you use HTML tags directly in Markdown, it can sometimes lead to unexpected behavior or formatting issues because Markdown processors may not handle HTML tags consistently or may have limitations on what HTML elements they support.

If you're encountering bugs or unexpected behavior when using HTML tags in a README.md file, here are a few things you can try:

Check Markdown Rendering: Ensure that the platform or tool you're using to render the Markdown supports the HTML elements you're using. Some Markdown parsers might have restrictions or limitations on HTML elements for security reasons or compatibility.

Use Markdown Syntax: Whenever possible, try to use Markdown syntax instead of HTML tags. Markdown is simpler and more widely supported across various platforms and tools.

Escape HTML: If you must use HTML tags, make sure to properly escape them. For example, instead of <div> use &lt;div&gt;. This can help prevent Markdown parsers from interpreting HTML tags as Markdown syntax.

Test and Iterate: Test your README file on different platforms and Markdown parsers to see how it renders. Make adjustments as needed to ensure consistent and correct rendering.

Consider Alternatives: If HTML tags are causing too many issues, consider using alternative Markdown elements or finding alternative ways to achieve the desired formatting without relying on HTML.

**Comment by IAmVigneswaran** [comment]
@smartdev328 Thanks for the reply!

Really hope, GitHub Team could fix the issue. Before February, it used to work.

Any idea on how I can fix this issue without changing the layout? I prefer keeping icon left aligned. 
https://github.com/TheAcharya/csv2notion-neo

Thanks.
